Include/internal/pycore_typeobject.h

```diff
@@ -109,6 +109,20 @@ extern static_builtin_state * _PyStaticType_GetState(PyInterpreterState *, PyTyp
 extern void _PyStaticType_ClearWeakRefs(PyInterpreterState *, PyTypeObject *type);
 extern void _PyStaticType_Dealloc(PyInterpreterState *, PyTypeObject *);
 
+PyAPI_FUNC(PyObject *) _PyType_GetDict(PyTypeObject *);
```

Comment:
This shouldn't be exposed to user code, as it bypasses the protections in `PyType_Modified()`

Reply:
I'll get this squared away.  Thanks for pointing it out.